zach groves add config to enable trace annotation analytics (#8699)
DataDog/documentation: content/en/tracing/app_analytics/_index.md

  }
}
```

> **Note:** If using [dd.trace.methods][8] or [trace annotations][9], app analytics for those spans can be enabled by setting `-Ddd.trace-annotation.analytics.enabled=true` .

Reviewer: ```suggestion
**Note:** App analytics for [dd.trace.methods][8] or [trace annotations][9] spans can be enabled by setting `-Ddd.trace-annotation.analytics.enabled=true`.

[8]: https://docs.datadoghq.com/tracing/custom_instrumentation/java/#dd-trace-methods
[9]: https://docs.datadoghq.com/tracing/custom_instrumentation/java/#trace-annotations
```

[5]: https://app.datadoghq.com/apm/settings
[6]: /tracing/visualization/#trace
[7]: /account_management/billing/apm_distributed_tracing/

> [8]: https://docs.datadoghq.com/tracing/custom_instrumentation/java/#dd-trace-methods

Reviewer: ```suggestion
```

> [9]: https://docs.datadoghq.com/tracing/custom_instrumentation/java/#trace-annotations

Reviewer: ```suggestion
```
